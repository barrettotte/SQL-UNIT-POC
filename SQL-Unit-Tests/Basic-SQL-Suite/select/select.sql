-- This is a simple test for a SELECT * statement --

SELECT * FROM [BARRETT_TEST].[dbo].[users]
