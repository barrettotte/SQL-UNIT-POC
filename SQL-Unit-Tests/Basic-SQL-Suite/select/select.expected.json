{
    "success": "true",
    "columns": [
        {
            "name": "id",
            "type": "BIGINT"
        },
        {
            "name": "first_name",
            "type": "VARCHAR(50)"
        },
        {
            "name": "last_name",
            "type": "VARCHAR(50)"
        },
        {
            "name": "username",
            "type": "VARCHAR(50)"
        }
    ],
    "rows": [
        {
            "first_name": "barrett",
            "last_name": "otte",
            "username": "barrettotte"
        },
        {
            "first_name": "some",
            "last_name": "body",
            "username": "somebody"
        },
        {
            "first_name": "hello",
            "last_name": "world",
            "username": "helloworld123"
        }
    ]
}